### PR TITLE
Index cleanup

### PR DESCRIPTION
**Pull request by tcharding**
Spikes repository has become stale.  

- `index.md` is out of date.
- There is a file name/number clash.
- There is a broken link

Fix these issues and move the newly fixed up contents of index into the README file.

Please note; `README.md` is used not `README.adoc`.  Can change if required.

### CODE REVIEW

**Review by luckysori — APPROVED** [review]
Cool! Thank you :smile:

File: README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+List of spike outcomes for COMIT network.
+
+- [spike-0000](0000-use-madr-like-solution-for-spike-outcomes.md) - Use MADR-like solution for spike outcomes - coblox/docs#36
+- [spike-0001](0001-basic-expiry-model.md) - Propose basic model to calculate expiry times - comit-network/RFCs#14
+- [spike-0002](0002-ether-htlc-dynamic-final-addresses.md) - Ether HTLC with dynamic final addresses
+- [spike-0003](0003-lightning-as-alpha-ledger.adoc) - LND as Alpha Ledger as described in RFC003
+- [spike-0004](0004-siren-prototype-accept-decline.adoc) - Siren Prototype for comit-rs HTTP API - Accept/Decline actions
+- [spike-0005](0005-kzen-two-party-ecdsa.adoc) - Kzen Two Party ECDSA Review
+- [spike-0006](0006-scriptless-scripts-with-ecdsa.adoc) - Scriptless Scripts with ECDSA
+- [spike-0008](0007-secret-seed-storage-and-key-derivation.md) - Secret seed storage and key derivation
```
Comment (da-kami) [inline README.md]:
```suggestion
- [spike-0007](0007-secret-seed-storage-and-key-derivation.md) - Secret seed storage and key derivation
```
Note: it seems spike `007` is actually not merged yet, see: https://github.com/comit-network/spikes/ - should this line be removed for now?

**Review by da-kami — CHANGES_REQUESTED** [review]
Nice! Added one comment with change request.

@D4nte should be able to decide if it should be `README.md` or `README.adco`, as far as I know we wanted to switch to Asciidoc.

File: README.md (hunk 1 above)
Comment (D4nte) [inline README.md]:
```suggestion
- [spike-0007](0007-secret-seed-storage-and-key-derivation.md) - Secret seed storage and key derivation
```

**Review by D4nte — APPROVED** [review]
Looks good. Please fix the typo.

MarkDown is fine for the readme IMHO.

File: README.md (hunk 1 above)
Comment (thomaseizinger) [reply README.md]: It is merged but as a number clash with `0003`.

**Review by thomaseizinger — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (tcharding) [reply README.md]: Fixed, thanks.

**Review by da-kami — APPROVED** [review]

**Pull request merged**